docs/installation/installation-using-docker-on-production-server.md

```diff
@@ -269,7 +269,7 @@ There are named volumes created under path `/var/lib/docker/volumes/` and one pe
 We create persisted folder with correct owner id `82` so internal docker `php-fpm` container user has access into the folder.
 ```
 mkdir /var/www/production-content
-chown -R 82:82 /var/www/production-content
+chown -R 33:33 /var/www/production-content
```

Comment:
good point, we have already debian but be careful because nginx is still with alpine. But maybe there is root(0). We will chek it during testing